tema-flex/watch.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"  a href="./style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>APPLE WATCH SERIES</h1>
+    <h4>Tehnologie avansată și stil într-un ceas inteligent </h6>
+        <br>
+        <br>
+        <br>
+    <div class="container">
+        <div class="watch">
+            <img src="./wda.jpg" alt="applewatch">
```

Comment:
```suggestion
            <img src="./da.jpg" alt="applewatch">
```

imaignea se numeste da.jpg